src/components/Popups/__tests__/PopupReply-test.tsx

```diff
@@ -28,11 +28,10 @@ describe('components/Popups/PopupReply', () => {
         preventDefault: jest.fn(),
         stopPropagation: jest.fn(),
     };
-    const mockTextarea = { focus: jest.fn(), value: '' };
     const getWrapper = (props = {}): ShallowWrapper => shallow(<PopupReply {...defaults} {...props} />);
 
     beforeEach(() => {
-        jest.spyOn(React, 'useRef').mockReturnValueOnce({ current: mockTextarea });
+        jest.spyOn(React, 'useState').mockImplementation(() => [EditorState.createEmpty(), jest.fn()]);
```

Comment:
It seems like PopupReply shouldn't be aware of draft-js at all. Can we mock out the entire ReplyField component?